portfolio/src/main/webapp/index.html

```diff
@@ -4,16 +4,142 @@
     <meta charset="UTF-8">
```

Comment:
This comment isn't a note about your code:

I'd recommend picking a higher contrast text color for the information on top of the image on the top of the page. Right now, it's white text on a light background, which can be hard to read for some people with visual impairments. I recommend using contrast checkers for accessibility to test if the contrast is high enough:

https://webaim.org/resources/contrastchecker/?fcolor=FFFFFF&bcolor=FF3258